src/modules/returnToNCI/returnToNCI-bar.js

```diff
@@ -300,7 +311,7 @@ var merge = function() {
                 // fancybox uses fixed elements for the lightbox feature.
                 // fixed elements of a transformed parent become relative to the parent instead of the viewport
                 //var sitename = document.querySelectorAll('script[src*=sitename]')[0].src.split("sitename=")[1];
-                return !!(document.getElementById('sidr-close') || document.head.innerHTML.match(/sidr/g) !== null) || settings.hasModalPopup || settings.hasSIDR || settings.hasFixedHeader || settings.hasFixedElements
+                return !!(document.getElementById('sidr-close') || document.head.innerHTML.match(/sidr/g) !== null) || settings.hasModalPopup || settings.hasSIDR || settings.hasFixedHeader || settings.hasFixedElements || settings.uxWindow || settings.sameUxWindow
```

Comment:
`settings.uxWindow` and `settings.sameUxWindow` should not be in this return statement. This function just checks if the site should use transform to animate the top bar or margin. It's unrelated to the window targets.

Reply:
Also, what exactly is settings.sameUxWindow???  There should be only 1 setting for uxWindow.

Reply:
Bryan, the second setting is for the ability to open them all in the same window. That's what I was discussing with you yesterday morning. Setting that one to true would set the target to "linkbacknewwindow" instead of "_blank". Volker and I thought being consistent with booleans was better than having one be a boolean or a string.

Reply:
It is common practice to default unset variables to false.  So it is not really a boolean.